Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp

```diff
@@ -319,7 +320,10 @@ void WebKitProtocolHandler::handleGPU(WebKitURISchemeRequest* request)
     addTableRow(displayObject, "Screen work area"_s, makeString(rect.x(), ',', rect.y(), ' ', rect.width(), 'x', rect.height()));
     addTableRow(displayObject, "Depth"_s, String::number(screenDepth(nullptr)));
     addTableRow(displayObject, "Bits per color component"_s, String::number(screenDepthPerComponent(nullptr)));
-    addTableRow(displayObject, "DPI"_s, String::number(screenDPI()));
+    addTableRow(displayObject, "Font Scaling DPI"_s, String::number(fontDPI()));
+    auto* data = screenData(displayID.value_or(primaryScreenDisplayID()));
+    if (data)
+        addTableRow(displayObject, "Monitor DPI"_s, String::number(data->monitorDPI));
```

Comment:
Instead of using ScreenProperties directly here add a screenDPI in PlatformScreen and use it there like the other screen properties.

Reply:
OK, but it will need to be `double screenDPI(PlatformDisplayID screendisplayID);` because different displays may have different DPIs.